docs/api/api.md

```diff
@@ -131,7 +148,32 @@ The data storage APIs include file system APIs, for file system operations, and
 
 ## Connectivity
 
-### Network interface
+### Common APIs
+<table>
+<tbody>
+<tr>
+<td><a href="#ble">BLE</a></td>
+<td><a href="#nfc">NFC</a></td>
+<td><a href="#lorawan">LoRaWAN</a></td>
+</tr>
+</tbody>
+</table>
+
+### RTOS only supported APIs
+<table>
+<tbody>
+<tr>
+<td><a href="#network-interface">Network interface</a></td>
+<td><a href="#socket">Socket</a></td>
+</tr>
+<tr>
+<td><a href="#secure-socket">Secure socket</a></td>
+<td><a href="#dns">DNS</a></td>
+</tr>
+</tbody>
+</table>
+
+#### Network interface
```

Comment:
I was very confused by this "dupicate" section. I think it should be removed. 
You could write in each sub-section header whether the API is supported or not, for example:
#### Network interface (RTOS only)
etc.
Or use formatting (font colour, bold/regular etc.) to distinguish RTOS only APIs.